[2022-03-29 20:48:09,264.264]	 WARNING [	app.loginuser:55]	 Usuario y/o contrase√±a incorrecta Admin
[2022-03-31 16:19:43,603.603]	 ERROR [	app.log_exception:1457]	 Exception on /registeruser/ [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1701, in make_response
    raise TypeError(
TypeError: The view function for 'login.registeruser' did not return a valid response. The function either returned None or ended without a return statement.
[2022-03-31 16:20:07,625.625]	 ERROR [	app.log_exception:1457]	 Exception on /registeruser/ [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1701, in make_response
    raise TypeError(
TypeError: The view function for 'login.registeruser' did not return a valid response. The function either returned None or ended without a return statement.
